_episodes/04-loop.md

```diff
@@ -147,7 +147,7 @@ for variable in collection:
 
 Using the oxygen example above, the loop might look like this:
 
-![loop_image](../fig/loops_image.png)
+![Graph illustrating a variable 'char' being assigned the value of each character in the word 'oxygen' in turn and then being printed](../fig/loops_image.png)
```

Comment:
```suggestion
![Loop variable 'char' being assigned the value of each character in the word 'oxygen' in turn and then being printed](../fig/loops_image.png)
```

Reply:
What about changing `being` to `is` and clarifying that we're printing to the screen?

Reply:
Also, see https://github.com/swcarpentry/python-novice-inflammation/pull/844 where @magsol has also proposed `alt` text for this image as well.

Reply:
Sorry! I missed this one.

Reply:
@spikelynch would you like to make the changes that Maxim suggests?